README.md

```diff
@@ -48,6 +48,8 @@ It is recommended, though not required, that you create new Models every time yo
 
 ## Future work
 
+The ModMenu mod should be considered _unstable_ until future work is pared down and version 1.0 is released. Breaking API changes may occur in the pursuit of implementing these additional features.
```

Comment:
> until future work is pared down

I don't think feature additions should generally be a blocker of 1.0, but even so I don't believe "pared down" sufficiently clarifies what is blocking 1.0. My intent with that comment is so that we have an actionable list of priorities we can work on to stabilize the API; we don't want to be in a place where we are on 0.x forever so we need a path out